src/functions/template-tags/date.php

```diff
@@ -307,7 +314,7 @@ function tribe_get_start_date( $event = null, $display_time = true, $date_format
 		 * @param string  $start_date
 		 * @param WP_Post $event
 		 */
-		return apply_filters( 'tribe_get_start_date', tribe_format_date( $start_date, $display_time, $date_format ), $event );
+		return $start_dates[ $cache_key ] = apply_filters( 'tribe_get_start_date', tribe_format_date( $start_date, $display_time, $date_format ), $event );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: f8cd23dd18f6c1a787cbc5e5be19192e</sub>